### PR TITLE
Docs improvements

### PR DESCRIPTION
**Pull request by cane-lagerwaard**
### Please check if the PR fulfills these requirements ###

- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
Update docs

**Does this PR introduce a breaking change? (What changes might users need to make in their application due to this PR?)**
No, only docs are updated

**Other things to mention**
Please check https://github.com/deity-io/falcon-docs/tree/doc-updates for new docs

### CODE REVIEW

File: packages/falcon-ui/README.md [hunk 1]
```diff
@@ -13,4 +13,4 @@
 
 ## Docs
 
-Please visit https://falcon-ui.docs.deity.io for detailed documentation.
+If you would like to find more information please check out the official documentation website [https://falcon.deity.io/](https://falcon.deity.io/)
```
[the remark this answers is not in the document]
Comment (cane-lagerwaard) [reply packages/falcon-ui/README.md]: @Borales correct, that's falcon-ui.docs.deity.io. Maybe it should be ui.falcon.deity.io instead

File: examples/shop-with-blog/client/README.md [hunk 2]
```diff
@@ -1,66 +0,0 @@
-# Shop with Blog Example
```
Comment (Borales) [inline examples/shop-with-blog/client/README.md]: @cane-lagerwaard I would keep READMEs for examples, people should have something to start with, not just another link to check the docs somewhere else.

Comment (cane-lagerwaard) [reply examples/shop-with-blog/client/README.md]: @Borales makes sense

File: packages/create-falcon-app/README.md [hunk 3]
```diff
@@ -2,77 +2,4 @@
...
-
-### Using your backend services
-
-By default `shop-with-blog` example uses demo Magento 2 and WordPress servers provided by DEITY so you can check how DEITY Falcon works. If you cannot start the server part (from server folder) or you see any problems with server behavior please contact us via our [community slack channel](http://slack.deity.io). 
-
-If you want to connect the generated app to your own Magento 2 backend you will need to install [Falcon Magento2 module](https://github.com/deity-io/falcon-magento2-module) which delivers REST endpoints required by DEITY Falcon and change configuration file (placed in `server/config/default.json`) to point the application to the correct url.
-
-Also if you want to connect DEITY Falcon app to your WordPress server you'll have to to install [Falcon WordPress module](https://github.com/deity-io/falcon-wordpress-module) and also change configuration file (`server/config/default.json`) to point the application to the correct server.
+If you would like to find more information please check out the official documentation website [https://falcon.deity.io/](https://falcon.deity.io/)
```
Comment (Borales) [inline packages/create-falcon-app/README.md]:
```suggestion
If you would like to find more information please check out the official documentation website [https://falcon.deity.io/docs/getting-started/installation#create-falcon-app-cli-tool](https://falcon.deity.io/docs/getting-started/installation#create-falcon-app-cli-tool)
```

File: packages/falcon-client/README.md [hunk 4]
```diff
@@ -4,572 +4,4 @@ This is a `@deity/falcon-client` host for your shop.
...
-        {
-          // your <Component />
-        }
-      </FalconClientMock>,
-      document.createElement('div')
-    );
-  });
-});
-```
+If you would like to find more information please check out the official documentation website [https://falcon.deity.io/](https://falcon.deity.io/)
```
Comment (Borales) [inline packages/falcon-client/README.md]:
```suggestion
If you would like to find more information please check out the official documentation website [https://falcon.deity.io/docs/basics/falcon-client](https://falcon.deity.io/docs/basics/falcon-client)
```

File: packages/falcon-server-env/README.md [hunk 5]
```diff
@@ -2,90 +2,4 @@
...
-  }],
-  "extensions": [{
-    "package": "my-custom-falcon-extension",
-    "config": {
-      "api": "my-custom-api"
-    }
-  }]
-}
-```
+If you would like to find more information please check out the official documentation website [https://falcon.deity.io/](https://falcon.deity.io/)
```
Comment (Borales) [inline packages/falcon-server-env/README.md]:
```suggestion
If you would like to find more information please check out the official documentation website [https://falcon.deity.io/docs/advanced/falcon-server-env](https://falcon.deity.io/docs/advanced/falcon-server-env)
```

File: packages/falcon-server/README.md [hunk 6]
```diff
@@ -2,145 +2,4 @@
...
- * `typeDefs: object` - gql expression with type definitions - helpful especially when extension needs to extend a type defined by other extension
- * `schema: object` - GrahQL schema generated by [makeExecutableSchema()](https://www.apollographql.com/docs/graphql-tools/schema-stitching.html) or [makeRemoteExecutableSchema()](https://www.apollographql.com/docs/graphql-tools/remote-schemas.html). Passing that option will override `typeDefs` property
- * `resolvers: object` - map with resolvers required by defined data types
- * `context: function` - function that modifies GraphQL context passed to resolvers during execution
- * `dataSources: object` - map with data sources that can be used by resolvers
-
-All the values passed via `typeDefs` and `schema` properties will be merged using [Apollo's schema stitching mechamism](https://www.apollographql.com/docs/graphql-tools/schema-stitching.html)
+If you would like to find more information please check out the official documentation website [https://falcon.deity.io/](https://falcon.deity.io/)
```
Comment (Borales) [inline packages/falcon-server/README.md]:
```suggestion
If you would like to find more information please check out the official documentation website [https://falcon.deity.io/docs/basics/falcon-server](https://falcon.deity.io/docs/basics/falcon-server)
```

File: packages/falcon-i18n/README.md [hunk 7]
```diff
@@ -2,17 +2,4 @@
...
-    |-blog.json
-    |-...
-  |-...
-```
-
-## Provided languages
-* EN
-* NL
\ No newline at end of file
+If you would like to find more information please check out the official documentation website [https://falcon.deity.io/](https://falcon.deity.io/)
```
Comment (Borales) [inline packages/falcon-i18n/README.md]:
```suggestion
If you would like to find more information please check out the official documentation website [https://falcon.deity.io/docs/basics/falcon-client#internationalization](https://falcon.deity.io/docs/basics/falcon-client#internationalization)
```

File: packages/falcon-dev-tools/falcon-i18n-webpack-plugin/README.md [hunk 8]
```diff
@@ -2,20 +2,4 @@
 
 This webpack plugin allows to extend your custom internationalization resources by defaults translations.
 
-## Options
-
-- `mainSource: string` - relative path to your custom i18n resources which should extend defaults
```
Comment (Borales) [inline packages/falcon-dev-tools/falcon-i18n-webpack-plugin/README.md]: @cane-lagerwaard I cannot find this section on falcon.deity.io

File: packages/falcon-logger/README.md [hunk 9]
```diff
@@ -3,31 +3,4 @@ Utility tool used for logging in Falcon packages.
...
-**Logger.verbose(...args)** - uses verbose logging in winston
-
-**Logger.debug(...args)** - for logging on "debug" level
-
-**Logger.info(...args)** - for logging on "info" level
-
-**Logger.warn(...args)** - for logging on "warn" level
-
-**Logger.error(...args)** - for logging on "error" level
+If you would like to find more information please check out the official documentation website [https://falcon.deity.io/](https://falcon.deity.io/)
```
Comment (Borales) [inline packages/falcon-logger/README.md]:
```suggestion
If you would like to find more information please check out the official documentation website [https://falcon.deity.io/docs/advanced/falcon-logger](https://falcon.deity.io/docs/advanced/falcon-logger)
```

File: packages/falcon-ui/README.md (hunk 1 above)
Comment (Borales) [inline packages/falcon-ui/README.md]: @cane-lagerwaard even it's a temporary domain - we should point to it

**Pull request merged**